Source/Scene/Cesium3DTileset.js

```diff
@@ -984,51 +988,60 @@ define([
                 } else {
                     // Tile does not meet SSE.
 
+                    // Get visibility for all children. Check if any visible children are not loaded.
+                    var allVisibleChildrenLoaded = true;
```

Comment:
Could be a `PERFORMANCE_TODO` but I wonder if we can exploit temporal coherence to burn a bit less CPU here.